content/en/tracing/configure_data_security/_index.md

```diff
@@ -13,31 +13,69 @@ aliases:
 ---
 ## Overview
 
-The performance data and traces that you're collecting with Datadog can contain sensitive information that you want to filter out, obfuscate, scrub, filter, modify, or just not collect. Additionally, it may contain synthetic traffic that might cause your error counts to be inaccurate or Datadog to not accurately indicate the health of your services.
+The Tracing Library collects data from an instrumented application that are sent to Datadog as traces. Instrumentations may collect data like personally identifiable information. If you are ingesting sensitive data as traces into Datadog, remediations can be added at ingestion with [Sensitive Data Scanner][12]. You can also configure the Datadog Agent or the Tracing Library to remediate sensitive data at collection before traces are sent to Datadog.
```

Comment:
```suggestion
Datadog tracing libraries collect data from an instrumented application. That data is sent to Datadog as traces and it may contain sensitive data such as personally identifiable information (PII). If you are ingesting sensitive data as traces into Datadog, remediations can be added at ingestion with [Sensitive Data Scanner][12]. You can also configure the Datadog Agent or the tracing library to remediate sensitive data at collection before traces are sent to Datadog.
```